freshdesk-import/holo_auth_freshdesk_import/main.py

```diff
@@ -0,0 +1,50 @@
+import os
+import requests
+import time
+
+CLOUDFLARE_ACCOUNT_ID = '18ff2b4e6205b938652998cfca0d8cff'
+CLOUDFLARE_WHITELIST_ID = '3130e2ca2d234214ba9db91622f7f197'
+FRESHDESK_COMPANY_ID = 36000445120
+
+
+def freshdesk_api_key():
+    return os.environ['FRESHDESK_API_KEY']
+
+
+def freshdesk_contacts(page):
+    return requests.get(
+        'https://holo.freshdesk.com/api/v2/contacts',
+        auth=(freshdesk_api_key(), '_'),
+        params={'company_id': FRESHDESK_COMPANY_ID, 'page': page}).json()
```

Comment:
```suggestion
        params={'company_id': FRESHDESK_COMPANY_ID, 'page': page, 'per_page':100}).json()
```
Makes us less likely to hit the API rate limit (default `per_page` is 10)

Reply:
Default `per_page` is actually 30. I'll try this out.

Reply:
That's what I meant :smile:

Reply:
Nah, sorry to say their API ignores the param. It still returns 30 per page even with `'per_page': 100` set.

Reply:
ah, at least we tried.